sublime_jedi/helper.py

```diff
@@ -67,16 +151,45 @@ def render(self, view, docstring):
         if is_sublime_v2():
             docstring_panel(view, docstring)
         else:
-            docstring_tooltip(view, docstring, simple_html_builder)
+            docstring_tooltip(view, docstring)
 
 
 class SublimeJediSignature(PythonCommandMixin, sublime_plugin.TextCommand):
-    """
-    Show signature in status bar
-    """
+    """Show signature in status bar."""
+
     def run(self, edit):
         ask_daemon(self.view, self.show_signature, 'signature')
 
     def show_signature(self, view, signature):
         if signature:
             sublime.status_message('Jedi: {0}'.format(signature))
+
+
+class SublimeJediTooltip(sublime_plugin.EventListener):
+    """EventListener to show jedi's docstring tooltip."""
+
+    def enabled(self):
+        """Check if hover popup is desired."""
+        return get_plugin_settings().get('enable_tooltip', True)
+
+    def on_activated(self, view):
+        """Handle view.on_activated event."""
+        if not self.enabled():
+            return
+        if not view.match_selector(0, 'source.python'):
+            return
+        # disable default goto definition popup
+        view.settings().set('show_definitions', False)
+
+    def on_hover(self, view, point, hover_zone):
+        """Handle view.on_hover event."""
+        if hover_zone != sublime.HOVER_TEXT:
+            return
+        if not self.enabled():
+            return
+        if not view.match_selector(point, 'source.python - string - comment'):
```

Comment:
I saw this function, but didn't realize the file being imported by helper already. I normally avoid importing single functions from modules. This way it is always clear where a function comes from. But this is a question of taste.